### PR TITLE
simplify ZAP pipeline interactions with S3

### PR DESCRIPTION
**Pull request by afeld**
The pipeline was previously downloading the results from all projects—this instead downloads the results from the particular project that is being scanned only. It also means we can use the built-in S3 resource, instead of our custom one.

### CODE REVIEW

**Comment by DavidEBest** [comment]
Is this [WIP]? What else needs to be done?

**Comment by afeld** [comment]
Testing, I suppose. Not sure if I'll get back to it today.

File: pipelines/zap/pipeline.yml [hunk 1]
```diff
@@ -74,14 +67,31 @@ jobs:
...
+          image_resource:
+            type: docker-image
+            source:
+              repository: busybox
+          outputs:
+          - name: last-results
+          run:
+            path: echo
+            args:
+            - Creating empty directory for results from the last scan.
```
Comment (afeld) [inline pipelines/zap/pipeline.yml:87]: This is a bit of a hack, but it works. We will just need to watch out for our scans reporting "NEW SITE" over and over, which means there's something wrong fetching the files.

**Comment by afeld** [comment]
Ready for another review!

**Comment by ctro** [comment]
Looks good to me!  I haven't tested locally yet though.

**Comment by ctro** [comment]
Works great for me locally, merging.

**Pull request merged**